docker/appserver/Tomcat/test/Dockerfile

```diff
@@ -5,16 +5,20 @@ FROM iaf-as-tomcat AS iaf-test-tomcat
 COPY src/scripts/catalinaAdditionalTest.properties /tmp
 RUN cat /tmp/catalinaAdditionalTest.properties >> /usr/local/tomcat/conf/catalina.properties  && rm -f /tmp/catalinaAdditionalTest.properties
 
+# copy war
+COPY target/dependencies/ibis-adapterframework-test.war /usr/local/tomcat/webapps/iaf-test.war
+
 # copy any additional drivers
 COPY target/dependencies/*.jar /usr/local/tomcat/lib/
 
-# copy database and jms configuration
-COPY target/dependencies/frank/context/context.xml /usr/local/tomcat/conf/Catalina/localhost/iaf.xml
+# provide database credentials
+RUN echo "testiaf_user"   > /opt/frank/secrets/testiaf_user/username
+RUN echo "testiaf_user00" > /opt/frank/secrets/testiaf_user/password
```

Comment:
Ik mis alleen nog de plek waar het alias gebruikt wordt.

Reply:
Die is er nog niet.